velox/exec/MergeJoin.cpp

```diff
@@ -222,6 +222,7 @@ void MergeJoin::addInput(RowVectorPtr input) {
   if (joinTracker_) {
     joinTracker_->resetLastVector();
   }
+  leftInputRows_ += input_->size();
```

Comment:
Why do we need leftInputRows_ and rightInputRows_. We can tell it by observing previous operator's output row count?